### DISCUSSION TITLE
When subscribed to data changes and then read a value from the OPC-UA server, it will continue to receive subscribed data even if its value has not changed

### DISCUSSION THREAD

**Discussion by anganing** [eclipse-milo/milo#1193] [General]
**Describe the bug**
When subscribed to data changes and then read a value from the OPC-UA server, it will continue to receive subscribed data even if its value has not changed. opc-ua server is s7-1200.

**Expected behavior**
I only want to receive a subscription when the value changes

**Logs and Packet Captures**
As long as I annotate this line to read the code, there will be no problem, but I must read the data when subscribing to data changes
![image](https://github.com/eclipse/milo/assets/63218247/f39e1dbe-880d-4283-abbd-b1e751a3f946)

**Additional context**
Version of Eclipse Milo you are using 0.6.3/0.6.11/0.6.10 all try. opc-ua server is simens s7-1200

**Comment by kevinherron** [comment]
I am not able to reproduce this with an S7-1200. 

You'll have to provide a Wireshark capture or otherwise prove it's not your code or the server at fault.
